static/styles/settings.css

```diff
@@ -1873,3 +1873,13 @@ body:not(.night-mode) #settings_page .custom_user_field .datepicker {
         max-height: unset;
     }
 }
+
+#edit_filter_pattern,
+#edit_filter_url_format_string {
+    width: 400px;
+}
+
+#edit-filter-pattern-status,
+#edit-filter-format-status {
+    margin-top: 10px;
+}
```

Comment:
Same with these; s/filter/linkifier` would be well worth doing in this PR just to avoid future churn -- we're planning to eliminate `realm_filters` from the lexicon.

Also, I'd probably prefer to not mix `_`s and `-`s for naming IDs for the same feature.